app/controllers/admin/administrators/administrators_controller.rb

```diff
@@ -61,6 +61,17 @@ def update
         end
       end
 
+      def delete
+        authorize @administrator, :destroy?
```

Comment:
This no longer needs to be :destroy?

Reply:
In fact, it doesn't need to be here or in destroy at all. It's in load_admin

Reply:
Just updated - removed completely as we have it on Application policy